developer-docs-site/docs/concepts/coin-and-token/aptos-token.md

```diff
@@ -155,14 +161,87 @@ The token-related data are stored at both creator’s account and owner’s acco
 | Royalty | Specifies the denominator and numerator for calculating the royalty fee. It also has the payee account address for depositing the royalty. |
 | `PropertyValue` | Contains both value of a property and type of property. |
 
-### Resource stored at the owner’s address
+#### Resource stored at the owner’s address
 
 | Field | Description |
 | --- | --- |
 | `TokenStore` | The main struct for storing the token owned by this address. It maps `TokenId` to the actual token. |
 | `Token` | The amount is the number of tokens. |
 | `TokenId` | `TokenDataId` points to the metadata of this token. The `property_version` represents a token with mutated `PropertyMap` from `default_properties` in the `TokenData`. |
 
+### Field descriptions
+
+The following tables contain descriptions and examples on each token field.
+
+#### Coin module key struct
+
+##### [`Coin`](https://github.com/aptos-labs/aptos-core/blob/744c2def47cddced878fda9bbd5633022fdb083a/aptos-move/framework/aptos-framework/sources/coin.move#L68)
+
+| Field | Type | Description |
+| --- | --- | --- |
+| value | u64 | Value of the token, eg: 1000000000 |
+
+##### [`CoinInfo`](https://github.com/aptos-labs/aptos-core/blob/744c2def47cddced878fda9bbd5633022fdb083a/aptos-move/framework/aptos-framework/sources/coin.move#L92)
+
+| Field | Type | Description |
+| --- | --- | --- |
+| name | String | Name of the token, eg: Aptos Coin |
+| symbol | String | Symbol for the token, eg: APT |
+| decimals | u8 | Determines how the value of coin is represented; for example APT’s decimal is 8, so a value of 100000000 is represented by 1 APT coin |
+| supply | Option&lt;OptionalAggregator&gt; | option::some(optional_aggregator::new(MAX_U128, parallelizable)) |
+
+#### Token module key struct
+
+##### [`Token`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move#L144)
```

Comment:
would it be possible to have a table to unify the tables describing struct and the table describing the fields in the structs
Struct | Description | Field | Type | Field Description

Reply:
Let me try and get back to you with questions.

Reply:
I am suggesting instead we link to the reference materials from each struct reference for the structure, for example:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/doc/token.md#0x3_token_TokenData

What do you think?

Reply:
Thank you! All done. Please see:
https://deploy-preview-5331--aptos-developer-docs.netlify.app/concepts/coin-and-token/aptos-token#token-resources

And note I did not see an entry for `PropertyValue` for linking in:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/doc/token.md

We could still try and merge tables. But it could be confusing since we don't spell out the fields for all structs.

I say let the auto-generated docs handle that, and we focus upon describing those where confusion might exist.

Reply:
Since we want to link to the auto-generated struct doc, we probably don't need the field description below?

Reply:
May we keep it? I see some value in the text description you provide not found at:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/doc/token.md#0x3_token_Token